data/news/contra_costa.json

```diff
@@ -3,11 +3,221 @@
   "title": "Contra Costa County COVID-19 News",
   "home_page_url": "https://www.coronavirus.cchealth.org/health-services-updates",
   "items": [
+    {
+      "id": "https://www.coronavirus.cchealth.org/frequently-asked-questions",
+      "url": "https://www.coronavirus.cchealth.org/frequently-asked-questions",
+      "title": "Updated Frequently Asked Questions",
+      "date_published": "2020-09-08T00:00:00-07:00"
+    },
+    {
+      "id": "https://www.coronavirus.cchealth.org/frequently-asked-questions",
+      "url": "https://www.coronavirus.cchealth.org/frequently-asked-questions",
+      "title": "Updated Frequently Asked Questions",
+      "date_published": "2020-09-08T00:00:00-07:00"
+    },
```

Comment:
There seems to be a weird doubling-up of entries happening here. I’ve filed https://github.com/sfbrigade/data-covid19-sfbayarea/issues/123 to fix it, but am going to merge this for now anyway.